EcoSoapBankTests/FontTests.swift

```diff
@@ -13,7 +13,7 @@ class FontTests: XCTestCase {
 
     func testMuliFontsInitializeProperly() {
         // given
-        for style in UIFont.MuliStyle.allCases {
+        for style in UIFont.MuliTypeface.allCases {
```

Comment:
Ah, I changed this in my PR as well lol. Whoever merges second will probably have to do some merge conflict fixing before merging into master.